### DISCUSSION TITLE
can I ask if I can apply CMap for pyMupdf to solve  UniKS-UTF16-H encoding issue?

### DISCUSSION THREAD

**Discussion by mk-docenty** [pymupdf/PyMuPDF#3786] [Looking for help]
Hi,

I have a question, sometimes some Korean encodings ( /UniKS-UTF16-H) are broken with most of PDF viewers.
is it possibe if I can Apply CMap for pyMupdf?  https://github.com/adobe-type-tools/cmap-resources/tree/master

You can refer to this issue for more information https://github.com/pymupdf/PyMuPDF/issues/3785
it will solve my issue.

**Comment by JorjMcKie** [comment]
You can try that - go ahead.

**Reply by mk-docenty** [reply]
I mean can I get some information how to do or if someone have done it?

**Comment by JorjMcKie** [comment]
Maybe some user will respond here.
PyMuPDF currently has no recipe for this.

**Reply by mk-docenty** [reply]
Thank you for quick response, I will keep an eye
